py/float.go

```diff
@@ -394,6 +394,21 @@ func (a Float) M__ge__(other Object) (Object, error) {
 	return NotImplemented, nil
 }
 
+// Properties
+func init() {
+	FloatType.Dict["is_integer"] = MustNewMethod("is_integer", func(self Object) (Object, error) {
+		if a, ok := convertToFloat(self); ok {
+			f, err := FloatAsFloat64(a)
+			if err != nil {
+				return nil, err
+			}
+			b := math.Abs(f - math.Round(f))
+			return NewBool(b < math.SmallestNonzeroFloat64), nil;
```

Comment:
Is there any reason to use this logic?
CPython check the float is an integer by using the floor API.
``` python
 o = (floor(x) == x) ? Py_True : Py_False;
```

Reply:
I calculated the difference between rounded value and original value.
If the value was less than the minimum value of float64, I think that it would be an integer.

I'll check the Cpython and then fix that part by using floor API.

Reply:
FYI, https://github.com/python/cpython/blob/279f44678c8b84a183f9eeb85e0b086228154497/Objects/floatobject.c#L853